member_card/demo/cooperators.xml

```diff
@@ -6,7 +6,6 @@
 <odoo>
     <record id="res_partner_cooperator_1_demo" model="res.partner">
```

Comment:
We should use demo data from the standard modules rather than adding new fake partners.